python-cli/mft_cli/airavata_mft_cli/bootstrap.py

```diff
@@ -62,15 +64,51 @@ def stop_service(bin_path, daemon_script_name):
   finally:
     os.chdir(current_dir)
 
+
+def check_java_version_method(required_version):
+  """
+      Issue 96: https://github.com/apache/airavata-mft/issues/96
+
+      References:
+      ---------
+      https://stackoverflow.com/questions/31807882/get-java-version-number-from-python
+      https://stackoverflow.com/questions/1332598/how-to-determine-whether-java-is-installed-on-a-system-through-python
+  """
+  if shutil.which("java"):
+    res = check_output(['java', '-version'], stderr=STDOUT).decode('utf-8')
```

Comment:
It is initially unclear what the following for loop does to fetch the java version. I think it is better to add an example output for "res" in a comment. Something like "#openjdk version "18.0.2" 2022-07-19"